<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>845</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Guardian Passwords</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>padlock.ico</normaloff>padlock.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(132, 0, 193, 255), stop:1 rgba(0, 204, 97, 255));
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1111</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame{
	background-color: rgb(255, 142, 144);
	border: 2px solid black;
	border-radius: 5px;
}
QLineEdit{
	border: 2px solid black;
	border-radius: 5px;
}
QPushButton:hover{
	background-color: rgb(206, 34, 34);
}
QPushButton:pressed{
	background-color: rgb(1, 35, 255);
}
QPushButton{
	background-color: rgb(255, 107, 102);
	border: 2px solid black;
	border-radius: 5px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labels">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Service:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_service">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2l">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Login:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_login">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3p">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_pass">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="add_record">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add Record</string>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="modify">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>80</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Modify</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="profil">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>80</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Profil</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>1110</width>
     <height>640</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
